apps/bot/src/commands/Help.ts

```diff
@@ -0,0 +1,23 @@
+import { Message } from 'discord.js';
+import { Command } from '../Command';
+import { BotClient } from '../types';
+
+export default class Help extends Command {
+  constructor(client: BotClient) {
+    super(client, {
+      name: 'help',
+      description: 'Help commands bot.',
+      category: 'Information',
+      usage: client.settings.prefix.concat('help'),
+      cooldown: 1000,
+      requiredPermissions: ['SEND_MESSAGES'],
+    });
+  }
+
+  public async run(message: Message): Promise<void> {
```

Comment:
Please fmt with `nx format`. This reminds me I need to add it as a pre-commit hook